pkg/auth/authz.go

```diff
@@ -0,0 +1,30 @@
+package auth
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/openshift/assisted-service/pkg/ocm"
+	"github.com/sirupsen/logrus"
+	"gorm.io/gorm"
+)
+
+type Ownership interface {
+	OwnedBy(ctx context.Context, db *gorm.DB) *gorm.DB
+}
+type Authorizer interface {
+	Ownership
+	CreateAuthorizer() func(*http.Request) error
+}
```

Comment:
Why separate these?

Reply:
For clarity

Reply:
I don't think it helps anything unless there's an actual use for the second interface, just keep one please.

Reply:
Fixed